desktop/core/src/desktop/lib/botserver/views_tests.py

```diff
@@ -72,63 +72,145 @@ def test_handle_on_message(self):
       handle_on_message("channel", None, "hello hue test", "user_id")
       assert_true(say_hi_user.called)
 
-  def test_handle_on_link_shared(self):
+  def test_handle_query_history_link(self):
```

Comment:
FYI: beware the test methods starts to test a lot of unrelated things (e.g. unfurl, check user perms, etc..)

Reply:
Removed some redundant unrelated checks and made a separate Slack user not Hue user test